### ISSUE TITLE
Info − Wiki.js works on FreeBSD

### ISSUE THREAD

**Issue by symsymmas** [Requarks/wiki#451]
Hi,
Not a problem but an info.
I use the stable release and it works on FreeBSD.
I give you this info because I found it neither on wiki.js.org nor on github. :)

**Comment by NGPixel** [comment]
Thanks for letting us know! Wiki.js should work on any platform where Node.js can run.

**Comment by blagoeres** [comment]
Hi, I am having problems running it on FreeBSD 11.2. I managed to make it working on all major Linux distros, but no luck on FreeBSD. Wiki.js just won't start for some reason. When I run `netstat -f inet -na | grep -i LISTEN` I don't see `127.0.0.1:3000` in the list which is the default port for Wiki.js, but `node wiki start` says `✔ Wiki.js has started successfully.` Any idea what can be problem ?

This is what I used to install it.

`sudo pkg install node8 npm-node8`

`sudo pkg install -y mongodb36`

`sudo pkg install -y nginx`

Configured Nginx according to Wiki.js docs.
